.buildkite/supported_plugins_test_pipeline.yml

```diff
@@ -21,8 +21,6 @@ steps:
 
   - group: "Tier2 plugins test group"
     key: "tier2_plugins"
-    # Wait all tier1 plugins testing ends before going on tier2
-    depends_on: "tier1_plugins"
```

Comment:
if we don't want to wait for the previous group to finish, do we still want to split them in groups, or just have them as separate steps?

Reply:
discuss this offline with @roaksoax , we prefer keeping them in separate groups for UI reasons.